cli/cmd_root.go

```diff
@@ -39,6 +39,7 @@ func init() {
 
 	// buildCmd
 	buildCmd.Flags().Bool("dummy", false, "Create a dummy bobfile")
+	buildCmd.Flags().Bool("no-cache", false, "Make true to disable save and load target from the cache")
```

Comment:
"do not use cache"

Reply:
Updated the description to "Set to true to not use cache"